global-styles.css

```diff
@@ -0,0 +1,53 @@
+:root{
+    --clr-white: #ffffff;
+    --clr-black: #000000;
+    --clr-grey: #808080;
+    --off-white: #e7e7e7;
+    --cart-blue: #5252eb;
+    --wishlist-red: #d84848;
+    --bg-color: #f6f6f6;
+    --text-secondary:#616161 ;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+}
```

Comment:
You may want to include `box-sizing: border-box` in the CSS reset, though it can be your preference.